lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -86,6 +86,19 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: { type: 'integer' },
+            maximumRetryAttempts: { type: 'integer' },
+          },
+        },
+        deadLetterConfig: {
+          type: 'object',
+          properties: {
+            arn: { type: 'string' },
```

Comment:
Does it have to be `arn` as a string or would it also support CF intrinsic functions e.g. `Ref` or `GetAtt` ?

Reply:
Here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-deadletterconfig.html only `Arn` as string value is mentioned, so pretty sure GetAtt will work as usual, but Ref will return the SQS name not the Arn and thus fail?